pkgs/development/tools/misc/saleae-logic/default.nix

```diff
@@ -9,38 +9,31 @@
 { stdenv, fetchurl, unzip, glib, libSM, libICE, gtk2, libXext, libXft
 , fontconfig, libXrender, libXfixes, libX11, libXi, libXrandr, libXcursor
 , freetype, libXinerama, libxcb, zlib, pciutils
-, makeDesktopItem, xkeyboardconfig
+, makeDesktopItem, xkeyboardconfig, dbus
 }:
 
 let
 
   libPath = stdenv.lib.makeLibraryPath [
     glib libSM libICE gtk2 libXext libXft fontconfig libXrender libXfixes libX11
     libXi libXrandr libXcursor freetype libXinerama libxcb zlib stdenv.cc.cc.lib
+    dbus
   ];
 
 in
 
+assert stdenv.system == "x86_64-linux";
```

Comment:
stdenv.system is deprecated since NixOS 18.09 (e.g. see 2c2f1e37d4374ea61caefd9389927ea03df4ce31). Here we should replace it with stdenv.hostPlatform.system.

Reply:
Done